v202207-1.md

```diff
@@ -39,8 +39,8 @@
 ### Security
 
 1. The [External Vault](https://www.terraform.io/enterprise/requirements/data-storage/vault) policy has been updated to use specific API paths instead of wildcard matching.
-1. The version of the internally-managed Nomad server has been updated to to 1.3.1.
-1. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies.
+2. The version of the internally-managed Nomad server has been updated to to 1.3.1.
+3. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies. This change bumps the version of Fluent Bit in `tfe-fluent-bit` to use 1.9.5.
```

Comment:
It's okay to leave the items as `1.`. The markdown rendering will give it real numbers.

```suggestion
1. The version of the internally-managed Nomad server has been updated to to 1.3.1.
1. Container updates have been adopted, addressing reported vulnerabilities (CVEs) in underlying packages / dependencies. This change bumps the version of Fluent Bit in `tfe-fluent-bit` to 1.9.5.
```